packages/react/src/components/numberInput/NumberInput.tsx

```diff
@@ -120,6 +120,13 @@ export const NumberInput = React.forwardRef<HTMLInputElement, NumberInputProps>(
       }
     }, [inputRef, ref]);
 
+    const dispatchNativeOnChangeEvent = (): void => {
```

Comment:
This is some higher-level DOM coding 😁 👍 
Do you know the reason why stepDown- and stepUp-method do not trigger native onChange?

Reply:
No idea. I made an issue about it to facebook/react: https://github.com/facebook/react/issues/22212